dms_version/models/storage.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017-2020 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+import bsdiff4
```

Comment:
```suggestion
import logging

from odoo import _, api, fields, models
from odoo.exceptions import ValidationError

_logger = logging.getLogger(__name__)
```

'bsdiff4' imported but unused